psscriptanalyzer/PSScriptAnalyzerSettings.psd1

```
@@ -0,0 +1,49 @@
+        'PSReviewUnusedParameter',
+        'PSProvideCommentHelp',
+        'PSAvoidTrailingWhitespace',
+        'PSUseConsistentIndentation',
+        'PSUseConsistentWhitespace',
+        'PSAvoidUsingPositionalParameters',
+        'PSAvoidUsingDoubleQuotesForConstantString'
+    )
+
+    Rules = @{
+        PSUseConsistentIndentation = @{
+            Enable          = $true
```

PSScriptAnalyzer (1.25.0)

[warning] Use space before and after binary and assignment operators.

Suggested fix: Use space before and after binary and assignment operators.

(PSUseConsistentWhitespace)